### ISSUE TITLE
Create unit and integration tests for plugin

### ISSUE THREAD

**Issue by foragerr** [jmeter-gradle-plugin/jmeter-gradle-plugin#16]

**Comment by djangofan** [comment]
Once you create some basic tests, I will probably be able to help a lot in this area of things.

**Comment by foragerr** [comment]
Gradle 2.6rc1 brings TestKit: https://docs.gradle.org/2.6-rc-1/userguide/test_kit.html
Should be useful to create tests

**Comment by djangofan** [comment]
yeah, that looks hopeful.  It sounds like using Spock is the way to go so that we get the object mocking  when/then stuff right?  https://code.google.com/p/spock/wiki/SpockBasics

**Comment by djangofan** [comment]
@foragerr  Looks like Gradle TestKit is now out...
https://docs.gradle.org/current/userguide/test_kit.html
I'm definitely interesting in helping write these tests, as time allows...
